src/components/Button/Button.tsx

```diff
@@ -5,7 +5,7 @@ import { Icon } from '../Icon/Icon';
 export interface Props {
   /**
    * Visually hidden button text (but text is still accessible to assistive technology).
-   * This overrides `text`
+   * This overrides `children` for screen readers
```

Comment:
Added "for screen readers" because I thought this might be confusing for people who are new to accessibility.